andela/ah-frontend-realers
# #166187813 Setup Sass pre-processing for Project

#### What does this PR do?

Set up Sass Pre-processing for project

#### Description of Task to be completed?

Install node-sass and sass-loader
Install css-loader and style-loader
Configure Sass in webpack.config file

#### How should this be manually tested?

* -	Git fetch this branch in the terminal with `git fetch origin ch-setup-sass-166187813
* -	Install application dependencies: `npm install`
* -	Run the application: `npm start`
* -       The app should show this:

<img width="418" alt="Screenshot 2019-07-05 at 17 47 08" src="https://user-images.githubusercontent.com/32802973/60730308-7c680200-9f4d-11e9-847a-3fceb89bd624.png">

* -	You can go to the `app.scss` file and add in any css code and you will see it effected in the browser. The above image shows blue text because this is what the `app.scss` file contains.

<img width="418" alt="Screenshot 2019-07-05 at 17 35 14" src="https://user-images.githubusercontent.com/32802973/60730476-e8e30100-9f4d-11e9-83d1-a19cf50337bd.png">

#### Any background context you want to provide?

This application did not have a sass pre-processors set up but I have set it up

#### What are the relevant pivotal tracker stories?

#[166187813](https://www.pivotaltracker.com/story/show/166187813)
